client/wfnews-war/src/main/angular/src/app/components/draggable-panel/draggable-panel.component.ts

```diff
@@ -762,4 +827,40 @@ type = 'evac-order';
       return date.toLocaleDateString('en-US', options);
     } else return '';
   }
+
+  extractPolygonData(response) {
+    const polygonData = [];
+    response.forEach(element => {
```

Comment:
Not super important, but typically it's better to use `for(const thing in array)` than `array.forEach` both of performance and scope reasons